cmd/transaction.go

```diff
@@ -20,12 +23,12 @@ func transactionCmdChain() *cobra.Command {
 		Short: "lookup a transaction for a given channel by transaction id",
 		RunE: func(cmd *cobra.Command, args []string) error {
 
-			client, err := mazzaroth.NewMazzarothClient(mazzaroth.WithAddress(viper.GetString(address)))
+			client, err := mazzaroth.NewMazzarothClient(mazzaroth.WithAddress(viper.GetString(channelAddress)))
 			if err != nil {
 				return err
 			}
 
-			receipt, err := client.TransactionLookup(cmd.Context(), viper.GetString(channelId), viper.GetString(transactionid))
+			receipt, err := client.TransactionLookup(cmd.Context(), viper.GetString(channelId), viper.GetString(transactionId))
```

Comment:
Should probably change this variable to "transaction"

Reply:
pushed